backend/prompt_studio/prompt_studio_registry/prompt_studio_registry_helper.py

```diff
@@ -241,6 +268,17 @@ def frame_export_json(
             llm = ""
             embedding_model = ""
 
+        if invalidated_prompts:
+            raise InValidCustomToolError(
+                f"Cannot export tool. Prompt(s) : {invalidated_prompts} "
```

Comment:
NIT: Can you try and pretty print the lists `invalidated_prompts` and `invalidated_outputs`. This will help avoid printing something like 
```
Cannot export Tool. Prompt(s): ['Test_prompt_1', 'Test_prompt_2'] do not have a valid prompt
``` 
and instead have
```
Cannot export Tool. Prompt(s): 'Test_prompt_1', 'Test_prompt_2' are not valid. Please enter a valid prompt.
```